src/modules/webhooks/stripe-webhook.ts

```diff
@@ -503,7 +502,9 @@ function stripePaymentIntentEventToPartialTransactionEventReportMutationVariable
         amount: paymentIntent.amount,
         currency: paymentIntent.currency,
       });
-      const type = TransactionEventTypeEnum.AuthorizationAdjustment;
+      const type = manualCapture
+        ? TransactionEventTypeEnum.AuthorizationSuccess
+        : TransactionEventTypeEnum.AuthorizationAdjustment;
```

Comment:
This one is a bit tricky - if this is the first Authorization, then `AuthorizationSuccess` event is fine, but [in case the Authorization amount was incremented](https://docs.stripe.com/payments/incremental-authorization) we might need to use `AuthorizationAdjustment` event type. This is because Saleor doesn't allow updating the Authorization any other way (`AuthorizationSuccess` with different amount than previously reported event would result in an exception).

`manualCapture` from my understanding, won't be a gauge whether this was a first authorzation, or if it was incremented. According to Stripe docs we should check `amount_updates` on `Charge` object to determine this.